libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -144,6 +146,7 @@ internal object MapboxNavigationTelemetry :
             "mapbox-navigation-android"
         }
         metricsReporter = reporter
+        feedbackEventCacheMap.clear()
```

Comment:
we have a `private fun reset()` to reset data on `init` and on `sessionStop`.
do we need to move `feedbackEventCacheMap.clear()` to `reset()` ?
or it shouldn't be cleaned until new `initialize ` / `postCachedUserFeedback `?

Reply:
> it shouldn't be cleaned until new initialize / postCachedUserFeedback 

that's correct.